lib/config.rb

```diff
@@ -0,0 +1,18 @@
+module Acts
+  module Permalink
+    class Config
+      attr_reader :to, :from, :separator, :max_length
+
+      def initialize(options={})
+        @config = options.with_indifferent_access
+
+        @to = @config[:to].try(:to_sym) || :permalink
+        @from = @config[:from].try(:to_sym) || :title
+        @separator = @config[:underscore] ? "_" : "-"
+
+        @max_length = @config[:max_length].to_i rescue 0
+        @max_length = 60 unless @max_length && @max_length > 0
```

Comment:
I think `@max_length && @max_length > 0` can just be `@max_length > 0`. The preceding line guarantees an integer is present, right?